### PR TITLE
Api/index option/data

### PR DESCRIPTION
**Pull request by ghost**
Option data from api/index

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -429,49 +429,49 @@ type: api
...
 - **Tipo:** `Object | Function`
 
-- **Restriction:** Only accepts `Function` when used in a component definition.
+- **Restricción:** Sólo acepta `Function` cuando es usado en la definición del componente.
 
-- **Details:**
+- **Detalles:**
 
-  The data object for the Vue instance. Vue will recursively convert its properties into getter/setters to make it "reactive". **The object must be plain**: native objects such as browser API objects and prototype properties are ignored. A rule of thumb is that data should just be data - it is not recommended to observe objects with their own stateful behavior.
+  El objeto de datos para la instancia de Vue. Vue convertirá recursivamente sus propiedades en getters/setters para hacerlas reactivas. **El objeto debe ser plano**: los objetos nativos como cada objeto de la API del navegador y propiedades proptotipadas son ignoradas. Una regla sobre el thumb es que estos datos solo deberían ser datos - no es recomendado observar objetos con sus propios comportamientos de estado.
```
Comment (FLarra) [inline src/v2/api/index.md]: What about translating?

```suggestion
  El objeto de datos para la instancia de Vue. Vue convertirá recursivamente sus propiedades en getters/setters para hacerlas reactivas. **El objeto debe ser plano**: los objetos nativos como cada objeto de la API del navegador y propiedades proptotipadas son ignoradas. Una regla de oro es que estos datos solo deberían ser datos - no es recomendado observar objetos con sus propios comportamientos de estado. 
```

File: src/v2/api/index.md [hunk 2]
```diff
@@ -429,49 +429,49 @@ type: api
...
+  El objeto de datos para la instancia de Vue. Vue convertirá recursivamente sus propiedades en getters/setters para hacerlas reactivas. **El objeto debe ser plano**: los objetos nativos como cada objeto de la API del navegador y propiedades proptotipadas son ignoradas. Una regla sobre el thumb es que estos datos solo deberían ser datos - no es recomendado observar objetos con sus propios comportamientos de estado. 
 
-  Once observed, you can no longer add reactive properties to the root data object. It is therefore recommended to declare all root-level reactive properties upfront, before creating the instance.
+  Una vez observado, usted puede no volver a adicionar propiedades a la raíz del objeto de datos. Se recomienda por lo tanto declarar todos las propiedades reactivas por adelantado a nivel de raíz, antes de de crear la instancia.
```
Comment (FLarra) [inline src/v2/api/index.md:438]: A couple of typos here:

```suggestion
  Una vez observado, usted puede no volver a adicionar propiedades a la raíz del objeto de datos. Se recomienda por lo tanto declarar todas las propiedades reactivas por adelantado a nivel de raíz, antes de crear la instancia. 
```

File: src/v2/api/index.md [hunk 3]
```diff
@@ -429,49 +429,49 @@ type: api
...
 
-  Once observed, you can no longer add reactive properties to the root data object. It is therefore recommended to declare all root-level reactive properties upfront, before creating the instance.
+  Una vez observado, usted puede no volver a adicionar propiedades a la raíz del objeto de datos. Se recomienda por lo tanto declarar todos las propiedades reactivas por adelantado a nivel de raíz, antes de de crear la instancia. 
 
-  After the instance is created, the original data object can be accessed as `vm.$data`. The Vue instance also proxies all the properties found on the data object, so `vm.a` will be equivalent to `vm.$data.a`.
+  Después que se crea la instancia se puede acceder al objeto original de los datos como `vm.$data`. La instancia de Vue también actúa como proxie a todas las propiedades encontradas en el objeto de los datos, así que `vm.a` será el equivalente a `vm.$data.a`.
```
Comment (FLarra) [inline src/v2/api/index.md]: I think we should translate `proxies` to `proxy`, like:

```suggestion
  Después que se crea la instancia se puede acceder al objeto original de los datos como `vm.$data`. La instancia de Vue también actúa como proxy a todas las propiedades encontradas en el objeto de los datos, así que `vm.a` será el equivalente a `vm.$data.a`.
```

File: src/v2/api/index.md [hunk 4]
```diff
@@ -429,49 +429,49 @@ type: api
...
 
-  Properties that start with `_` or `$` will **not** be proxied on the Vue instance because they may conflict with Vue's internal properties and API methods. You will have to access them as `vm.$data._property`.
+  La propiedades que comienzan con `_` o `$` **no** serán proxiadas en la instancia de Vue debido a que pueden causar conflicto con las propiedades internas de Vue y los métodos de la API. Usted tendrá que acceder a ellas como `vm.$data._property`.
```
Comment (FLarra) [inline src/v2/api/index.md]: Another typo here and minor suggestion: 

```suggestion
  Las propiedades que comienzan con `_` o `$` **no** serán parte del proxy de la instancia de Vue debido a que pueden causar conflicto con las propiedades internas de Vue y los métodos de la API. Usted tendrá que acceder a ellas como `vm.$data._property`.
```

What do you think?

File: src/v2/api/index.md [hunk 5]
```diff
@@ -429,49 +429,49 @@ type: api
...
 
-  When defining a **component**, `data` must be declared as a function that returns the initial data object, because there will be many instances created using the same definition. If we use a plain object for `data`, that same object will be **shared by reference** across all instances created! By providing a `data` function, every time a new instance is created we can call it to return a fresh copy of the initial data.
+  Al definir un **componente**, `data` debe ser declarada como una función que devuelve los datos del objeto inicial, porque habrán muchas instancias creadas usando la misma definición. Si usamos un objeto plano para `data`, este mismo objeto será **compartido por referencia** a lo través de todas las instancias creadas! Al proporcionar una función `data`, cada vez que una nueva instancia es creada podemos llamarla para devolver una copia reciente de los datos iniciales.
```
Comment (FLarra) [inline src/v2/api/index.md]: Typo here:

```suggestion
  Al definir un **componente**, `data` debe ser declarada como una función que devuelve los datos del objeto inicial, porque habrán muchas instancias creadas usando la misma definición. Si usamos un objeto plano para `data`, este mismo objeto será **compartido por referencia** a través de todas las instancias creadas! Al proporcionar una función `data`, cada vez que una nueva instancia es creada podemos llamarla para devolver una copia reciente de los datos iniciales.
```

File: src/v2/api/index.md [hunk 6]
```diff
@@ -429,49 +429,49 @@ type: api
...
+  // debe usar la funcion cuando esta en Vue.extend()
   var Component = Vue.extend({
     data: function () {
       return { a: 1 }
     }
   })
   ```
 
-  Note that if you use an arrow function with the `data` property, `this` won't be the component's instance, but you can still access the instance as the function's first argument:
+  Note que si usted usa una función flecha con la propiedad `data`, `this` no será la instancia del componente,  pero usted puede seguir accediendo a la instancia por el primer argumento de la función:
```
Comment (FLarra) [inline src/v2/api/index.md]: Extra whitespace here :) 

```suggestion
  Note que si usted usa una función flecha con la propiedad `data`, `this` no será la instancia del componente, pero usted puede seguir accediendo a la instancia por el primer argumento de la función:
```

**Comment by Tintef** [comment]
@iamalejandrofull You forgot to change the title `## Options / Data`

**Comment by sxcamacho** [comment]
@Tintef @FLarra  I'm going to merge this pr, but we need to continue working on this section (options/data)

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**